LICENSE.md

```diff
@@ -52,39 +52,36 @@ The User accepts the following liability:
  - This Mod may alter fundamental parts of the Game.
  - The User is liable for all damages resulting from the use or misuse of the Mod.
 
-3. Right to use
+3. Right to use and examine
```

Comment:
Using something and examining it are two very different things. 

As a User, I might never even think about the fact that the mod is written in Java. 

If I'm examining the mod for ideas, to learn about the Codebase, or to understand how the Author did something, I'm definitely falling under the "Use" definition, but it's a very specific use that I wanted to call out due to certain situations in the modding community that would have never happened under a license with wording like this.